test/Microsoft.DocAsCode.MarkdigEngine.Extensions.Tests/Microsoft.DocAsCode.MarkdigEngine.Extensions.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\Shared\test.base.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
is this a must-have for vulnerability fix?

Reply:
This change is for the second issue mentioned in the description. From the scan report, the error is from test\Microsoft.DocAsCode.MarkdigEngine.Extensions.Tests\obj\project.assets.json, so this should be the source. Is there any concern changing this?

Reply:
Nope! Thanks for explanation!

Reply:
I see. Can you also upgrade other projects with `netcoreapp2.0`?

Please also let @superyyrrzz know so that we can add this info into release note.

Reply:
Talked to @superyyrrzz, there are only two test projects using netcoreapp2.0. Other projects are using the shared framework defined in base.props. Already cover these two projects here.